docs/devguide/building/index.rst

```diff
@@ -527,3 +529,171 @@ Users of OpenDDS are strongly encouraged to select one of the two options listed
 If this is not possible, users of OpenDDS must make sure that all code generator, compiler, and linker settings in the custom build setup result in API- and ABI-compatible code.
 To do this, start with an MPC or CMake-generated project file (makefile or Visual Studio project file) and make sure all relevant settings are represented in the custom build system.
 This is often done through a combination of inspecting the project file and running the build with verbose output to see how the toolchain (code generators, compiler, linker) is invoked.
+
+.. _cmake-building:
+
+****************************
+Building OpenDDS Using CMake
+****************************
+
+.. versionadded:: 3.26
+
+OpenDDS can be built with CMake 3.23 or later.
+
+Configuring and Building
+========================
+
+.. ifconfig:: is_release
+
+  If not already done, download the source from :ghrelease:`GitHub`.
+
+To configure and build:
+
+.. tab:: Linux, macOS, BSDs, etc.
+
+  .. code-block::
+
+    cmake -B build -DCMAKE_UNITY_BUILD=TRUE
+    cmake --build build -- -j 4
+
+  4 was used as an example for the max number of parallel jobs.
+  If unsure what this number should be, use the number of CPU cores on the machine.
+  This can be combined with unity builds.
+
+.. tab:: Windows
+
+  .. code-block::
+
+    cmake -B build -DCMAKE_UNITY_BUILD=TRUE
+    cmake --build build
```

Comment:
These are very similar.  Could they be refactored to just show the commands with an optional `[-- <build tool args>]` part at the end?

Reply:
They could, but I don't really want to force someone to have to try to piece together what the right commands are, especially if they're not familiar with them.

Reply:
Just a philosophical point about a "cross-platform make" (CMake).  We could also leave out the parallel build part and let that be an exercise for the reader.